### ISSUE TITLE
Permissions Issue

### ISSUE THREAD

**Issue by RichShumaker** [indieweb/wordpress-indieweb#9]
I have added the plug in BUT
I am getting a permissions issue when I try to install
"You do not have sufficient permissions to access this page."

What permissions level do I need to set and where as I can get to the files to modify the permissions.

I was able to get all the dependencies loaded and I had to update to PHP higher than 5.3 for that so I am glad this came up as I needed to be on a higher version of PHP just didn't know I wasn't.

Thanks for the help.
Rich Shumaker

**Comment by pfefferle** [comment]
Hey @RichShumaker, you need to be loggedin with "administrator" rights.

**Comment by RichShumaker** [comment]
I am and I still have the issue.  To be clear I am logged into WordPress and I am the Admin of that WordPress which is on my server and that error popped up when I tried to install the plug in.

That is why I was asking if I need to change the permissions on any specific files.
Thanks.
Rich Shumaker

**Comment by dshanske** [comment]
Could be the permissions on the plugin directory.

**Comment by pfefferle** [comment]
@RichShumaker are you trying to install it through the plugin manager of wordpress or have you copied the files by hand?

**Comment by RichShumaker** [comment]
I am using the plugin manager of wordpress.  I apologize I did not say that in the original post.

**Comment by pfefferle** [comment]
@RichShumaker can you re-try it with the latest changes?

**Comment by RichShumaker** [comment]
@pfefferle Sorry for the HUGE delay.  I tried again and it didn't work.
So I deactivated and deleted a few of the components and then reloaded the plug in.  It said it was 5 months old from the plug in guide and after installing it I had the same issue.
I am using Add New Plug In and typing Indieweb and it comes up but as I said above it is 5 months old.  Do I need to somehow update the list? I thought that would be automatic but I would like to force it to update to grab the newest version.

Thanks again for the help.
Rich Shumaker
